tox.ini

```diff
@@ -29,7 +29,7 @@ max-line-length = 120
 
 [pytest]
 python_files =
-;    test_*.py
+    test_*.py
```

Comment:
@Seenivasanseeni I had commented this out, so that we can focus on 1 particular module at one time. But nvm. 👍